jax/_src/lax/control_flow/loops.py

```diff
@@ -155,10 +158,6 @@ def scan(f, init, xs, length=None):
   dtype (or a nested tuple/list/dict container data structure with a fixed
   structure and arrays with fixed shape and dtype at the leaves).
 
-  .. note::
-    :py:func:`scan` compiles ``f``, so while it can be combined with
-    :py:func:`jit`, it's usually unnecessary.
-
```

Comment:
Drive-by: I removed these whilst I was editing the docstrings, as right now JIT'd code is the only fast path, so I think this advice is contrary to good practice.